tensorflow/python/autograph/converters/control_flow.py

```diff
@@ -309,9 +309,8 @@ def visit_For(self, node):
     node = self.generic_visit(node)
     body_scope = anno.getanno(node, annos.NodeAnno.BODY_SCOPE)
     iter_scope = anno.getanno(node, annos.NodeAnno.ITERATE_SCOPE)
-
     loop_vars, undefined, _ = self._get_block_vars(
-        node, body_scope.bound | iter_scope.bound)
+        node, body_scope.bound | iter_scope.bound, iter_scope.bound)
```

Comment:
I think it's a bit overkill to thread it through arguments. I'd say, just add the symbols to loop_vars directly here, after having called _get_block_vars.

Reply:
I've recovered the original API here.